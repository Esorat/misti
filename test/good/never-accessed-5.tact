fun test() {
    while(true) { let a: Int = 0; }
}
