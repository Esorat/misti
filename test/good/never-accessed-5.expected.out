test/good/never-accessed-5.tact:2:19:
  1 | fun test() {
> 2 |     while(true) { let a: Int = 0; }
                        ^
  3 | }
Read-only variable
Help: Consider creating a constant instead
See: https://nowarp.github.io/docs/misti/docs/detectors/ReadOnlyVariables